types/empower-core/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-redundant-jsdoc-2": false,
```

Comment:
Please suppress the rules on individual lines, rather than globally.

Reply:
Done